product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api-and-apply-qos/1.1-common/src/main/java/org/wso2/am/scenario/test/common/APIRequest.java

```diff
@@ -51,11 +53,26 @@ public void init() {
         this.addParameter("visibility", this.visibility);
         this.addParameter("version", this.version);
         this.addParameter("swagger", this.swagger);
+        this.addParameter("tags", this.tags);
```

Comment:
We are not passing tags, description as arguments in this constructor. So we shouldn't be initializing them here.